presto-main/src/test/java/com/facebook/presto/sql/query/TestGrouping.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.query;
+
+import org.testng.annotations.Test;
+
+public class TestGrouping
```

Comment:
This suggest that those might be all grouping tests. Consider renaming it to maybe: `TestGroupingWithoutTpch` (I am not sure really). I wonder how does this fit into greater effort to refactor tests: https://github.com/junit-pioneer/junit-pioneer/issues/62

Reply:
What's the current status of that refactor?

Reply:
Unfortunately I think it's stalled for now. @ebd2 stopped working on that then TD thing happened. We will see how free resources we will have for it.